package.json

```diff
@@ -3,19 +3,24 @@
   "type": "module",
   "scripts": {
     "dev": "vite -c vite.config.ts",
+    "dev:sw": "vite -c vite.config.sw.ts",
     "dev:ssr": "ts-node --experimentalSpecifierResolution node --esm server/server",
     "build": "npm run build:client",
+    "build:sw": "rm -rf dev-dist/sw && vite build --config vite.config.sw.ts",
     "build:client": "vite build",
     "build:ssr": "vite build -c vite.config.server.ts",
     "build:netlify": "vite build -c vite.config.netlify.ts"
   },
   "dependencies": {
+    "@spryker-oryx/presets": "latest",
     "@spryker-oryx/themes": "latest",
-    "@spryker-oryx/presets": "latest"
+    "dexie": "^3.2.2",
```

Comment:
```suggestion
    "dexie": "^3.0.0",
```

if possible

Reply:
Oh, it's actually can be removed, I've used it for testing.

Reply:
it is still there

Reply:
Removed.